diff.py

```diff
@@ -965,10 +965,11 @@ def serialize_data_references(references: List[Tuple[int, int, str]]) -> str:
 def maybe_get_objdump_source_flags(config: Config) -> List[str]:
     flags = []
 
-    if config.show_line_numbers or config.source:
+    if config.show_line_numbers or config.show_source or True:
```

Comment:
Doesn't this make the condition always true and cause `--line-numbers` and `--source` to be always passed? If so, why not just remove the if altogether?

That said, unconditionally passing those source flags is *highly* undesirable for larger projects (e.g. BotW). As mentioned last time we discussed the `show_line_numbers` default, objdump + --source takes an unacceptable amount of time for interactive diffing of larger functions: 3s+ for a 30kB function, and 42s on an extreme case (150kB function).

I'd prefer to have a --always-parse-source flag instead of forcing options that cause a dramatic slowdown for larger projects... (a significant time cost that just goes to waste if the source info isn't even printed in the end).

Reply:
Both changes in this fn were from testing, my mistake for including them (committed at the wrong time!)